stories/accordion/index.js

```diff
@@ -4,7 +4,6 @@ import { withDocs } from 'storybook-readme';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import { css } from '@emotion/core';
-
```

Comment:
We usually leave line breaks between dependency imports and file imports (and then another line break between relative path imports, like you see on line 11 below.)